lib/commands/nw-test/index.js

```diff
@@ -34,6 +36,37 @@ module.exports = {
     this.quickTemp.remove(this, '-testsDistNW');
   },
 
+  TestServerBuilder: Builder.extend({
```

Comment:
I'd love to see this `TestServerBuilder` section removed in favor of using addon hooks to manipulate the build pipeline. From what I understand, the reason why you're extending the builder is to do stuff after the build. I think this can be done more elegantly by modifying the addon's [`index.js`](https://github.com/brzpegasus/ember-cli-node-webkit/blob/v0.2.0/index.js) to have the following hook:

```js
postprocessTree: function(type, tree) {
  var funnel = require('broccoli-funnel');
  var mergeTrees = require('broccoli-merge-trees');
  var replace = require('broccoli-string-replace');

  if (type === 'all' && process.env.EMBER_ENV === 'test') {
    // Update the base URL in `tests/index.html`
    var index = replace(tree, {
      files: ['tests/index.html'],
      pattern: {
        match: /base href="\/"/,
        replacement: 'base href="../"'
      }
    });

    // Copy `tests/package.json` to the output directory
    var testPkg = funnel('tests', {
      files: ['package.json'],
      destDir: '/tests'
    });

    return mergeTrees([tree, index, testPkg], { overwrite: true });
  }

  return tree;
}
```